### PR TITLE
Add more applications

### PR DESCRIPTION
**Pull request by odow**
image

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -42,6 +40,7 @@ julia> x = solve_constrained_least_squares_regression(A, b)
 
 ## JuMP is used...
 
+ * to power exa-scale research into personalized medicine as part of [PerMedCoE](https://permedcoe.eu/core-applications/)
```
Comment (odow) [inline index.md:43]: cc @exaexa is this correct?

Comment (exaexa) [reply index.md]:
```suggestion
 * to accelerate pre-exascale metabolic modeling workflows in
   [PerMedCoE project](https://permedcoe.eu/core-applications/)
```

thanks!

**Comment by odow** [comment]
Update link to https://pubsonline.informs.org/doi/10.1287/trsc.2022.0342 cc @gdalle

File: index.md (hunk 1 above)
Comment (odow) [reply index.md]: What does `pre-exascale` mean?

File: index.md [hunk 2]
```diff
@@ -42,22 +40,32 @@ julia> x = solve_constrained_least_squares_regression(A, b)
 
 ## JuMP is used...
 
- * to solve large-scale [inventory routing problems at Renault](https://arxiv.org/abs/2209.00412),
-   [schedule trains at Thales Inc.](https://www.sciencedirect.com/science/article/pii/S0191261516304830),
-   [plan power grid expansion at PSR](https://juliacomputing.com/case-studies/psr/),
-   and
-   [route school buses](https://www.the74million.org/article/building-a-smarter-and-cheaper-school-bus-system-how-a-boston-mit-partnership-led-to-new-routes-that-are-20-more-efficient-use-400-fewer-buses-save-5-million/)
+ * to model and solve real-world problems, including
+    * large-scale inventory routing problems at [Renault](https://arxiv.org/abs/2209.00412)
```
Comment (odow) [inline index.md:44]:
```suggestion
    * large-scale inventory routing problems at [Renault](https://pubsonline.informs.org/doi/10.1287/trsc.2022.0342)
```

File: index.md [hunk 3]
```diff
@@ -42,22 +40,32 @@ julia> x = solve_constrained_least_squares_regression(A, b)
...
  * to build continental-scale energy system models, including
-     MIT's [GenX](https://github.com/GenXProject/GenX),
-     the National Renewable Energy Laboratory's [Sienna](https://www.nrel.gov/analysis/sienna.html),
-     and
-     Spine's [SpineOpt](https://www.tools-for-energy-system-modelling.org)
+     * MIT's [GenX](https://github.com/GenXProject/GenX)
+     * the National Renewable Energy Laboratory's [Sienna](https://www.nrel.gov/analysis/sienna.html)
+     * Spine's [SpineOpt.jl](https://www.tools-for-energy-system-modelling.org)
+     * TNO's [TulipaEnergyModel.jl](https://github.com/TulipaEnergy/TulipaEnergyModel.jl)
+ * to power exa-scale research into personalized medicine at
```
Comment (odow) [inline index.md:61]:
```suggestion
 * to accelerate pre-exascale metabolic modeling workflows at
```

**Pull request merged**